docs/_articles/en/v3.x/salesforce-graph-engine/working-with-sfge.md

```diff
@@ -82,6 +82,47 @@ Example:
 /* sfge-disable ApexFlsViolationRule */
 public class MyClass {
 ```
+
+## Unblock Graph Engine Analysis
+
+If your Graph Engine analysis is intentionally blocked, it’s because Graph Engine identified something incorrect in your code. You must modify your code to unblock the analysis. Depending on the situation, you see one of these messages.
+
+
+| Violation | When it Occurs | Message |
+| -------- | ------- | ------- |
+| User Action | Returned one time on an entire analysis. <br> Analysis of all code is blocked. | Remove unreachable code to proceed with the analysis. |
+| User Action Violation | Returned on a single path. <br> Analysis of only that code path is blocked. | Rename or delete this reused variable to proceed with the analysis. |
```

Comment:
This table renders a little weirdly, due to the linebreaks in the cells and the lack of explicit borders. Not sure what we can do about that. Just think it's worth noting.
![Screen Shot 2023-04-25 at 9 59 36 AM](https://user-images.githubusercontent.com/4054488/234318495-6d801ece-776e-40e3-bb01-b78d0a867890.png)